packages/ckeditor5-dev-release-tools/lib/tasks/verifypackagespublishedcorrectly.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const upath = require( 'upath' );
+const { globSync } = require( 'glob' );
+const fs = require( 'fs-extra' );
+const { checkVersionAvailability } = require( '../utils/checkversionavailability' );
+
+/**
+ * Npm sometimes throws incorrect error 409 while publishing, while the package uploads correctly.
+ * The purpose of the script is to validate if packages that threw 409 are uploaded correctly to npm.
+ *
+ * @param {Object} options
+ * @param {String} options.packagesDirectory Relative path to a location of packages to release.
+ * @param {String} options.version Version of the current release.
+ * @param {Function} options.onSuccess Callback fired when function is successful.
+ * @returns {Promise}
+ */
+module.exports = async function verifyPackagesPublishedCorrectly( options ) {
+	const { packagesDirectory, version, onSuccess } = options;
+	const packagesToVerify = globSync( upath.join( packagesDirectory, '*' ), { absolute: true } );
```

Comment:
Could we use `await glob()` instead? AFAIR, we aim to have all commands async to avoid blocking the `listr2` process.